agent/structs/structs_test.go

```diff
@@ -589,13 +589,6 @@ func TestStructs_NodeService_ValidateExposeConfig(t *testing.T) {
 			func(x *NodeService) { x.Proxy.Expose.Paths[0].ListenerPort = 65536 },
 			"invalid listener port",
 		},
-		"duplicate paths": {
```

Comment:
Wouldn't you rather keep the test case and change it to be a "this is explicitly allowed" style test instead?

Reply:
Yep, that would reflect the policy better. Updated.